### PR TITLE
Refactor Info and Accounts classes and bump the minor version.

### PR DESCRIPTION
**Pull request by dontmitch**

### CODE REVIEW

**Comment by dontmitch** [comment]
Sorry, I didn't intended for the last commit (which broke Travis) to get included with this PR - just removed it.

File: lib/plaid/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Plaid
-  VERSION = '1.4.3'
+  VERSION = '1.4.5'
```
Comment (michaelckelly) [inline lib/plaid/version.rb]: Should this be `1.5.0` since we're doing a minor version bump?

**Comment by michaelckelly** [comment]
@dontmitch this looks great, thanks for taking the time to put this together.

File: lib/plaid/version.rb (hunk 1 above)
Comment (dontmitch) [inline lib/plaid/version.rb]: Thanks for catching, my bad. I haven't dealt with versioning in a while :). I said bump the minor version in my commit message and then just bumped the patch.

**Comment by michaelckelly** [comment]
Going to go ahead and merge this - thanks again @dontmitch

**Pull request merged**